### PR TITLE
Survey info

### PR DESCRIPTION
**Pull request by OlaShabalina**
## Overview of the Pull Request:
Styled the 3rd component as per Figma design except for the background image.

**Desktop view**
image
**Mobile view**
image

## Yet to do
* Integrating real data into the design.
* Fixing the background image

I used the following code for the background:
```
backgroundImage={`url(${Background}) no-repeat`}
```
If you know how to fix it - please do 😄
Temporary solution is to have borders as lines.

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

File: pages/results.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import React from "react";
 import useForm from "../components/FormProvider";
 import Layout from "../components/Layout/Layout";
 import DownloadResults from "../components/ResultsPageComponents/DownloadResults/DownloadResults";
+import SurveyInfo from "../components/ResultsPageComponents/SurveyInfo/SurveyInfo";
```
Comment (hqtan) [inline pages/results.js:6]: Hi @OlaShabalina ,
What do you think about renaming `SurveyInfo` component to `SurveyOverview`, to align with the section heading?

```suggestion
import SurveyOverview from "../components/ResultsPageComponents/SurveyOverview/SurveyOverview";
```

**Review by hqtan — APPROVED** [review]
Awesome work @OlaShabalina ! 🙇🏽 

I only touched up a couple of typos;

Re the background image, we might not have to worry too much about that, if the designers are ok with a tiled border for the 4 Flex components.

I left a suggestion about renaming `SurveyInfo`; happy for you to merge it as-is.

da best

File: pages/results.js (hunk 1 above)
Comment (OlaShabalina) [reply pages/results.js:6]: Happy with that Qing! ;-)

**Comment by OlaShabalina** [comment]
Thanks @hqtan , we probably need to copy text into MS Word prior to adding to the page. I don't read the content ))))

**Pull request merged**